### PR TITLE
Licenses

### PR DESCRIPTION
**Pull request by alexeymorozov**
Added licenses.

Need help with getting licenses of these distributions:

1. DengXian https://github.com/fontist/formulas/issues/25
https://download.microsoft.com/download/4/0/9/4094ABA6-1A50-4FFD-A8CC-846BD66573F0/eadengfontpack.exe

2. Yu https://github.com/fontist/formulas/issues/24
https://gitlab.com/fontmirror/archive/-/raw/master/eayufontpack.exe

3. Segoe UI https://github.com/fontist/formulas/issues/29
https://web.archive.org/web/20170104231942if_/http://download.microsoft.com/download/F/8/8/F88CB355-ECAA-4B74-87D6-C05C81D215BF/ExcelViewer.exe

4. Adobe Reader 19 https://github.com/fontist/fontist/issues/127
http://ardownload.adobe.com/pub/adobe/reader/win/AcrobatDC/misc/FontPack1902120058_XtdAlf_Lang_DC.msi

5. Adobe Reader 20 https://github.com/fontist/fontist/issues/128
http://ardownload.adobe.com/pub/adobe/reader/win/AcrobatDC/2001220041/AcroRdrDC2001220041_en_US.exe

### CODE REVIEW

**Comment by ronaldtse** [comment]
Licenses for Adobe Reader 19 and 20:
https://supportdownloads.adobe.com/support/downloads/license.html

Will revert on the rest, can't find!

**Comment by ronaldtse** [comment]
The licenses for Office 2013 Language Packs:
https://www.microsoft.com/en-us/Useterms/OEM/Office/2013HomeAndBusiness/Useterms_OEM_Office_2013HomeAndBusiness_English.htm

**Comment by ronaldtse** [comment]
As suspected, there's really a license to accept for installing the MS fonts:

image

(http://blog.yottun8.com/archives/107)

**Comment by alexeymorozov** [comment]
Thanks for the tips! Added all licenses, including missing 1, 2, 3

File: Formulas/segoe_ui.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ display_progress_bar: true
 resources:
   Segoe_UI.exe:
     urls:
-    - https://web.archive.org/web/20170104231942if_/http://download.microsoft.com/download/F/8/8/F88CB355-ECAA-4B74-87D6-C05C81D215BF/ExcelViewer.exe
-    sha256: 56e2fcd583ffaaa316257de7b208e6be411c4e343b7e4072c95053baa11539af
+    - https://archive.org/download/ExcelViewer/ExcelViewer.exe
+    sha256: 4fc8e08237e8b458091c83dde68139e779fe401b4884d92d66ec843b5ca4a2ca
```
Comment (alexeymorozov) [inline Formulas/segoe_ui.yml:10]: The ExcelViewer package contained German version of license, so I've replaced a link with the English one. Though newer archive is larger (74 vs 52 MB).

Comment (ronaldtse) [reply Formulas/segoe_ui.yml:10]: That’s absolutely fine. The concern is whether the latest version of the font is included.

**Comment by ronaldtse** [comment]
Thanks @alexeymorozov !

**Comment by alexeymorozov** [comment]
Ok :) Versions are the same.

**Pull request merged**